### PR TITLE
Document Endpoint Security integration limitation

### PR DESCRIPTION
**Pull request by david-kow**

### CODE REVIEW

File: docs/orchestrating-elastic-stack-applications/agent-fleet.asciidoc [hunk 1]
```diff
@@ -15,7 +15,7 @@ This section describes how to configure and deploy Elastic Agent in link:https:/
 * <<{p}-elastic-agent-fleet-quickstart,Quickstart>>
 * <<{p}-elastic-agent-fleet-configuration,Configuration>>
 * <<{p}-elastic-agent-fleet-configuration-examples,Configuration Examples>>
-* <<{p}-elastic-agent-fleet-known-limitation,Known Limitation>>
+* <<{p}-elastic-agent-fleet-known-limitations,Known Limitation>>
```
Comment (barkbay) [inline docs/orchestrating-elastic-stack-applications/agent-fleet.asciidoc:18]:
```suggestion
* <<{p}-elastic-agent-fleet-known-limitations,Known Limitations>>
```

File: docs/orchestrating-elastic-stack-applications/agent-fleet.asciidoc [hunk 2]
```diff
@@ -435,10 +435,14 @@ kubectl apply -f {agent_recipes}/fleet-apm-integration.yaml
 
 Deploys single instance Elastic Agent Deployment in Fleet mode with APM integration enabled.
 
-[id="{p}-elastic-agent-fleet-known-limitation"]
-== Known limitation
+[id="{p}-elastic-agent-fleet-known-limitations"]
+== Known limitations
 
+=== Running as a root and within a single namespace
```
Comment (sebgl) [inline docs/orchestrating-elastic-stack-applications/agent-fleet.asciidoc:441]:
```suggestion
=== Running as root and within a single namespace
```

File: docs/orchestrating-elastic-stack-applications/agent-fleet.asciidoc [hunk 3]
```diff
@@ -435,10 +435,14 @@ kubectl apply -f {agent_recipes}/fleet-apm-integration.yaml
...
 Elastic Agent in Fleet mode has to run as a root, and in the same namespace as the Elasticsearch cluster it connects to.
```
Comment (sebgl) [inline docs/orchestrating-elastic-stack-applications/agent-fleet.asciidoc:442]:
```suggestion
Elastic Agent in Fleet mode has to run as root, and in the same namespace as the Elasticsearch cluster it connects to.
```

**Review by sebgl — APPROVED** [review]

**Review by barkbay — APPROVED** [review]

**Comment by david-kow** [comment]
Hey @alaudazzi, I'm merging this for now, let me know if you have any concerns, we can fix them in a separate PR.

**Pull request merged**